File: excercise-13/testFor.py
from datetime import datetime


def test_for(n):
    for i in range(n):
        #print(i)
        pass


if __name__ == "__main__":
    init = datetime.now()

    test_for(10000000)

    end = datetime.now()
    delta = end - init

    print(int(delta.total_seconds() * 1000))

File: excercise-13/testFor.go
package main

import (
	"fmt"
	"time"
)

func testFor(n int) {
	for i := 0; i < n; i++ {
		//fmt.Println(i)
	}
}

func main() {
	init := time.Now()

	testFor(10000000)

	end := time.Now()
	delta := end.Sub(init).Milliseconds()

	fmt.Println(delta)
}
